sccameraview/src/main/java/com/example/sccameraview/Camera2View.java

```diff
@@ -355,6 +366,41 @@ private static Size chooseVideoSize(Size[] choices) {
         return choices[choices.length - 1];
     }
 
+    private File getOutputMediaFile(int type){
```

Comment:
it is supposed to be a temporary solution to be able to test recording and taking a photo. File is saved based on which function was called